openshift4/templates/backend.dc.yaml

```diff
@@ -386,6 +386,26 @@ objects:
                     secretKeyRef:
                       key: database-nris-password
                       name: postgresql-nris
+                - name: COMMON_SERVICES_CLIENT_ID
```

Comment:
It won't let me comment on the line since it's out of the committed of changes, but if you change https://github.com/bcgov/mds/pull/1856/files#diff-e89a9cf81abf84592135e97e8b2e257f16b74e89c28aff020b987e605a2fb387R253
to ${ENVIRONMENT_NAME} it should pass through the value

Reply:
Good catch!